tests/files/no-unused-modules/cjs.js

```diff
@@ -0,0 +1,7 @@
+// Simple import extracted from 'redux-starter-kit' compiled file
+
+function isPlain(val) {
+  return typeof val === 'undefined' || val === null || typeof val === 'string' || typeof val === 'boolean' || typeof val === 'number' || Array.isArray(val) || isPlainObject(val);
```

Comment:
```suggestion
  return true;
```
There's no need to have actual logic here.